src/snmalloc/backend/fixedglobalconfig.h

```diff
@@ -117,8 +120,16 @@ namespace snmalloc
     {
       static_assert(B::wildness == capptr::dimension::Wildness::Wild);
 
-      static const size_t sz = sizeof(
-        std::conditional<std::is_same_v<std::remove_cv<T>, void>, void*, T>);
+      constexpr size_t sz = []() constexpr {
+        if constexpr (proxy::is_same_v<proxy::remove_cv_t<T>, void>)
+        {
+          return sizeof(void*);
+        }
+        else
+        {
+          return sizeof(T);
+        }
+      }();
 
```

Comment:
```suggestion
      static const size_t sz = sizeof(
        proxy::conditional_t<proxy::is_same_v<proxy::remove_cv<T>, void>, void*, T>);
```
Ah, I see, so shouldn't this really be a `conditional_t`, and then the rewrite is much smaller?

Reply:
I can have a try but I remember void cannot appear at the given position.

Reply:
Thanks

Reply:
It actually works. It did not work previously because it was using `remove_cv` instead of `remove_cv_t`.

I have promoted this fix to the second patch of the serious as it is more related to type traits.